src/components/dex/OrdersTable.vue

```diff
@@ -152,24 +146,23 @@ export default {
     },
 
     formatTableData(tableData) {
-      return tableData.reduce((formattedData, tableEntry) => {
-        const entry = {
+      return tableData.map(({ marketName, offerId, price, quantity, side }) => {
+        return {
           pairAndSide: {
-            pair: tableEntry.marketName,
-            side: tableEntry.side.toLowerCase(),
+            marketName,
+            side: side.toLowerCase(),
           },
           details: {
-            amount: tableEntry.quantity,
-            base: this.baseCurrency,
-            price: tableEntry.price,
-            cost: this.$formatNumber(tableEntry.price * this.baseCurrencyUnitPrice),
+            quantity,
+            quoteSymbol: _.last(marketName.split('-')),
```

Comment:
I’m pretty sure there is a field in the market for the quote symbol so you don’t have to split the market name.